frontend/package.json

```diff
@@ -12,7 +12,8 @@
     "dev": "yarn clean && yarn plugin-stats && ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode development --watch --progress",
     "build": "yarn clean && yarn plugin-stats && NODE_ENV=production ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/webpack --mode=production",
     "coverage": "jest --coverage .",
-    "lint": "eslint --ext .js,.jsx,.ts,.tsx --color .",
+    "lintme": "eslint --ext .js,.jsx,.ts,.tsx --color",
```

Comment:
I'd prefer to go with @christianvogt's suggestion:

> Adding a new eslint script without a path would be very flexible and solve your use case:
> `"eslint": "eslint --ext .js,.jsx,.ts,.tsx --color",`

Please replace `lintme` with `eslint`. All developers should use the same `eslint` script for unified experience.

The fact that `eslint` script in `package.json` over-shadows `./node_modules/.bin/eslint` delegation (`yarn run`) is not a problem, so no need to come up with alternative script names like `lintme`.